src/main/java/com/futurcollars/lesson8/zad2/Shape.java

```diff
@@ -0,0 +1,13 @@
+package com.futurcollars.lesson8.zad2;
+
+public abstract class Shape {
+
+    int getArea() {
+        return 0;
+    }
+
+    int getPerimeter() {
+        return 0;
+    }
```

Comment:
te metody mogą być abstrakcyjne, to wymusi ich zaplementowanie przez klasy dziedziczące ;)